docs/releases/1.10.txt

```diff
@@ -135,6 +135,8 @@ Forms
 Generic Views
 ^^^^^^^^^^^^^
 
+* Added request.user to the debug view
```

Comment:
- add double backticks around request.user since it's a code block
- add period at end of sentence
- move to "Requests and Responses" section